commcare_connect/form_receiver/processor.py

```diff
@@ -42,15 +42,16 @@
 
 def process_xform(xform: XForm, hq_server: HQServer):
     """Process a form received from CommCare HQ."""
-    app = get_app(xform.domain, xform.app_id, hq_server)
     user = get_user(xform)
 
-    opportunity = get_opportunity(deliver_app=app)
+    opportunity = get_opportunity(xform.domain, hq_server, deliver_app_id=xform.app_id)
```

Comment:
`get_opportunity` raises an error if multiple _active_ opps are found for same app.